Acme/Certificate/Formatter/FullChainFormatter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Bundle\Acme\Certificate\Formatter;
+
+use AcmePhp\Bundle\Acme\CertificateAuthority\Configuration\CertificateAuthorityConfigurationInterface;
+use AcmePhp\Core\Ssl\Certificate;
+use AcmePhp\Core\Ssl\KeyPair;
+
+/**
+ * Format the full chain certificate to the certificate authority.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class FullChainFormatter implements FormatterInterface
+{
+    /** @var CertificateAuthorityConfigurationInterface */
+    private $certificateAuthorityConfiguration;
+
+    /**
+     * Formatter constructor.
+     *
+     * @param CertificateAuthorityConfigurationInterface $certificateAuthorityConfiguration
+     */
+    public function __construct(CertificateAuthorityConfigurationInterface $certificateAuthorityConfiguration)
+    {
+        $this->certificateAuthorityConfiguration = $certificateAuthorityConfiguration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'fullchain.pem';
```

Comment:
This one is the fullchain.

I'll have a look deeper in the original acme CLI to double check